app/cocina_builders/contributor_role_cocina_builder.rb

```diff
@@ -8,24 +8,15 @@ class ContributorRoleCocinaBuilder # rubocop:disable Metrics/ClassLength
   }.freeze
 
   ROLES = {
+    # both organization and person roles
     author: {
       value: 'author',
       code: 'aut',
       uri: 'http://id.loc.gov/vocabulary/relators/aut',
       source: SOURCE
     },
-    advisor: {
-      value: 'advisor'
-    },
-    # person roles
-    composer: {
-      value: 'composer',
-      code: 'cmp',
-      uri: 'http://id.loc.gov/vocabulary/relators/cmp',
-      source: SOURCE
-    },
     contributing_author: {
-      value: 'contributing author',
```

Comment:
There were a handful of incorrect values fixed here a below. These should correspond with the MARC relator.